website/versioned_docs/version-2.0.0/user-guides/upgrade.md

```diff
@@ -0,0 +1,28 @@
+---
+id: upgrade
+title: Upgrade Litmus ChaosCenter to Latest Version
+sidebar_label: Upgrade
+---
+
+---
+
+To upgrade Litmus ChaosCenter from 2.0.0-Beta9 to 2.0.0, you can follow these steps:
+
+1. ## Control Plane Upgrade
```

Comment:
Are we sure about this - this will not take into account any rbac changes. Apply of manifest is better?

Reply:
Shall we add this @rajdas98 ?

Reply:
Reapply would be good then